packages/app/src/Pages/Login.tsx

```diff
@@ -279,7 +279,7 @@ export default function LoginPage() {
           <div className="flex">
             <input
               dir="auto"
-              type="text"
+              type="password"
```

Comment:
This doesnt really make sense when you want to login with the NIP-5 handle or the pubkey, maybe this should switch to password type when it starts with nsec

Reply:
Done!